erpnext/regional/report/uae_vat_201/uae_vat_201.html

```diff
@@ -0,0 +1,77 @@
+{%
+	var report_columns = report.get_columns_for_print();
+	report_columns = report_columns.filter(col => !col.hidden);
+%}
+<style>
+	.print-format {
+		padding: 10mm;
+		font-size: 8.0pt !important;
+		font-family: Tahoma, sans-serif;
+	}
```

Comment:
Any reason for adding these styles?

Reply:
Yes
Doesn't look proper without it
Used in other reports too